topics/basic.md

```diff
@@ -1,78 +1,110 @@
-[//]: # (title: Basic and form)
-
-<include src="lib.xml" include-id="outdated_warning"/>
+[//]: # (title: Basic authentication)
 
 <microformat>
-<p>Code examples:</p>
-<p><a href="https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-basic">auth-basic</a></p>
-<p><a href="https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-basic-hash-table">auth-basic-hash-table</a></p>
-<p><a href="https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-form">auth-form</a></p>
+<p>
+Required dependencies: <code>io.ktor:ktor-auth</code>
+</p>
+<p>
+Code examples: <a href="https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-basic">auth-basic</a>, <a href="https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-basic-hash-table">auth-basic-hash-table</a>
+</p>
 </microformat>
 
+The basic authentication scheme is a part of [HTTP framework](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication) used for access control and authentication. In this scheme, user credentials are transmitted as user name/password pairs encoded using Base64. 
+
+Ktor allows you to use basic authentication for logging in users and protecting specific [routes](Routing_in_Ktor.md). You can get general information about authentication in Ktor from the [](authentication.md) section.
+
+> Given that basic authentication passes user name and password as clear text, you need to use [HTTPS/TLS](ssl.md) to protect sensitive information.
+
 ## Add dependencies {id="add_dependencies"}
-To enable `basic` and `form` authentication, you need to include the `ktor-auth` artifact in the build script:
+To enable the `basic` authentication, you need to include the `ktor-auth` artifact in the build script:
 <var name="artifact_name" value="ktor-auth"/>
 <include src="lib.xml" include-id="add_ktor_artifact"/>
 
-## Usage {id="usage"}
+## Basic authentication flow {id="flow"}
 
-Ktor supports two methods of authentication with the user and raw password as credentials:
-`basic` and `form`.
+The basic authentication flow looks as follows:
```

Comment:
You don't need the article for basic. Please drop in all places.

Reply:
> The basic authentication flow

agree in a case of `basic authentication` but here `the` refers to flow